covid_model/sql/mobility_dwell_hours.sql

```diff
@@ -0,0 +1,7 @@
+select
```

Comment:
doesn't compute any normalization yet, since we need to aggregate up to regions first.